data/scripts.js

```diff
@@ -114,7 +114,7 @@ exports.BattleScripts = {
 		let movename = move.name;
 		if (move.id === 'hiddenpower') movename = 'Hidden Power';
 		if (sourceEffect) attrs += '|[from]' + this.getEffect(sourceEffect);
-		if (zMove && (move.category === 'Status')) {
+		if (zMove && (move.category === 'Status') && !move.isZ) {
```

Comment:
Have you tested this? Wouldn't `move.isZ` be true for all Z moves, not just special Z moves?